### PR TITLE
Split test suite into unit vs integration

### PR DESCRIPTION
**Pull request by phillc**

### CODE REVIEW

File: account_events.go [hunk 1]
```diff
@@ -4,10 +4,10 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"log"
-	"strconv"
-	"strings"
 	"time"
+
+	"github.com/linode/linodego/internal/parseabletime"
+	"github.com/linode/linodego/internal/utils"
```
Comment (displague) [inline account_events.go]: https://blog.golang.org/package-names#TOC_5.

Comment (phillc) [reply account_events.go]: good call

**Comment by displague** [comment]
How does renaming the test package affect the examples?
https://godoc.org/github.com/linode/linodego#pkg-examples

**Comment by phillc** [comment]
> How does renaming the test package affect the examples?
> https://godoc.org/github.com/linode/linodego#pkg-examples

Are you asking why the tests failed? Or are you saying there is an oversight on the package rename?

**Pull request merged**